packages/cf-component-card/src/CardContent.js

```diff
@@ -16,7 +17,7 @@ class CardContent extends React.Component {
         {this.props.children}
         { this.props.footerMessage ? (
           <div className="cf-card__footer_message">
```

Comment:
We can remove `className` though right?  If not I think we need to rename it because its not following the BEM naming structure.